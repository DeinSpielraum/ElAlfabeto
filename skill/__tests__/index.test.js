const skill = require('../index');

describe('Skill', () => {
  it('should invoke LaunchIntent', () => {
    return skill.invoke(require('../__mocks__/request.LaunchRequest.json'))
    .then(responseBody => expect(responseBody).toMatchSnapshot())
  });

  it('should invoke PracticeIntent', () => {
    return skill.invoke(require('../__mocks__/request.PracticeIntent.json'))
    .then(responseBody => console.log(responseBody))
  });
});
