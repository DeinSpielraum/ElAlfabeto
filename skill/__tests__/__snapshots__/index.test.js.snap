// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill should invoke LaunchIntent 1`] = `
Object {
  "response": Object {
    "card": Object {
      "content": "¡Hola! ¡Bienvenido al Alfabeto con MiKarmen! Puedo explicarte cómo vamos a aprender el alfabeto o podemos empezar directamente a practicar. ¿Qué quieres hacer? ¿Escuchar la explicación o empezar a practicar?",
      "title": "¡Bienvenido al Alfabeto con MiKarmen!",
      "type": "Simple",
    },
    "outputSpeech": Object {
      "ssml": "<speak>¡Hola! ¡Bienvenido al Alfabeto con MiKarmen! Puedo explicarte cómo vamos a aprender el alfabeto o podemos empezar directamente a practicar. ¿Qué quieres hacer? ¿Escuchar la explicación o empezar a practicar?</speak>",
      "type": "SSML",
    },
    "reprompt": Object {
      "outputSpeech": Object {
        "ssml": "<speak>¿Escuchar la explicación o empezar a practicar?</speak>",
        "type": "SSML",
      },
    },
    "shouldEndSession": false,
  },
  "sessionAttributes": Object {},
  "userAgent": "ask-node/2.0.7 Node/v8.11.1",
  "version": "1.0",
}
`;
